.github/workflows/vet.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: vet
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  vet:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.19'
+    - name: Install tools
+      run: |
+        go install golang.org/x/lint/golint@latest && \
+        go install golang.org/x/tools/cmd/goimports@latest && \
+        go install honnef.co/go/tools/cmd/staticcheck@latest
+    - name: Execute vet.sh
+      run: ./.github/workflows/vet.sh
```

Comment:
Should we just call the existing script instead and not make a copy or move it?

Reply:
I meant to delete the `internal/kokoro/vet.sh` file, moving it to `.github/workflows`, because it was going to be executed there instead of in Kokoro. I think I accidentally added it back in!

Where would like the file to exist, I don't really care.

Reply:
I've fixed it, so removed `internal/kokoro/vet.sh` in favor of `.github/workflows/vet.sh`.